### DISCUSSION TITLE
complete function without filling params snippet

### DISCUSSION THREAD

**Discussion by tydra-wang** [hrsh7th/nvim-cmp#1172] [Q&A] [answered]
How can I complete function without filling params snippet?
I just want to see the document window after completing a function call, but not the params definitions.

I want:
```go
   fmt.Printf(|)
```
I got:
```go
    fmt.Printf(|format string, a ...any)
```

**Comment by Shougo** [comment, marked as answer]
Please see this.

https://github.com/hrsh7th/nvim-cmp/issues/1129

**Reply by tydra-wang** [reply]
thanks!

this comment is what I want.
https://github.com/neovim/nvim-lspconfig/issues/276#issuecomment-651700649

**Comment by tydra-wang** [comment]
Finally I find out that `init_options.usePlaceholders` is responsible for this.
```lua
   nvim_lsp.gopls.setup {
      -- ...
      settings = {
         gopls = {
            experimentalPostfixCompletions = true,
            analyses = {
               unusedparams = true,
               -- shadow = true,
            },
            staticcheck = true,
         },
      },
      -- init_options = {
      --    usePlaceholders = true,
      -- }
   }

```
